src/db/user.ts

```diff
@@ -10,7 +12,9 @@ class User {
     }
 
     asString(): string {
-        return this.getUserName() + ", " + this.yearOfBirth + ", Pasha visited " + this.pavelDurovVisits + " times."
+        if (this.surname)
+            return this.getUserName() + " " + this.surname + ", " + this.yearOfBirth + ", Pasha visited " + this.pavelDurovVisits + " times."
+        else return this.getUserName() + ", " + this.yearOfBirth + ", Pasha visited " + this.pavelDurovVisits + " times."
```

Comment:
Notice how there is a lot of repeating code here. Can you think of a way to mitigate that?